src/scipp/plt/view.py

```diff
@@ -1,102 +1,22 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
 
-from .figure import Figure
-from .toolbar import Toolbar
-
-import ipywidgets as ipw
-
-
-class SideBar:
-
-    def __init__(self, children=None):
-        self._children = children if children is not None else []
-
-    def _ipython_display_(self):
-        return self._to_widget()._ipython_display_()
-
-    def _to_widget(self):
-        return ipw.VBox([child._to_widget() for child in self._children])
-
 
 class View:
```

Comment:
Is it meaningful to instantiate `View`? If not, it could inherit from `abc` and `notify` could be an abstract method.